docs/setup-guide/arktos-with-mizar-cni.md

```diff
@@ -0,0 +1,40 @@
+# Arktos to Enforce the Multi-tenant with Mizar Network Feature
+
+This document captures the steps applied to an Arktos cluster lab enabling the unique multi-tenant network feature. The machines in this lab used are AWS EC2 t2-large (2 CPUs, 8GB mem), Ubuntu 18.04 LTS.
+
+The steps might change with the progress of development.
+  
+1. Update Kernel
+To check kernel, run following command
+
+```bash
+uname -a
+```
+
+To update Kernel, download and run:
+If it is `5.6.0-rc2` then you can skip downloading ```kernelupdate.sh```
+```bash
+wget https://raw.githubusercontent.com/CentaurusInfra/mizar/dev-next/kernelupdate.sh
+bash kernelupdate.sh
+```
+
+2. Start Arktos cluster
+```bash
+CNIPLUGIN=mizar ./hack/arktos-up.sh
```

Comment:
When I tried this out with a freshly installed Ubuntu 18.04 VM in AWS and running kernelupdate with sudo, I got the following:
```
ubuntu@ip-192-168-1-15:~/arktos$ CNIPLUGIN=mizar ./hack/arktos-up.sh
WARNING : The kubelet is configured to not fail even if swap is enabled; production deployments should disable swap.
WARNING : This script MAY be run as root for docker socket / iptables functionality; if failures occur, retry as root.
Containerd is required for Arktos
ubuntu@ip-192-168-1-15:~/arktos$
```
What kind of testing was done here?

Reply:
You need to mention that user should run hack/setup-dev-node.sh after the kernelupdate step. If you had bothered to do some testing and tried it out, you would have hit these issues.

Reply:
Added the required steps in the guide and tested, Thanks

Reply:
Make sure you test this guide from the top with a fresh ubuntu 18.04 vm in AWS and on-prem and let me know when it is actually tested and ready. No more garbage. I don't want to waste time my again trying out untested crap.
It is a very basic , common-sense expectation that you test user facing documentation the same way a new user would try it.

Reply:
We have tested the complete guide and we will be sharing the successful test report in email